### ISSUE TITLE
Open Project Discussion Thread

### ISSUE THREAD

**Issue by droseblockchain** [CatholicBlockchainOrg/SacramentalRegistry#1]
We want to store sacramental records anonymously on the blockchain, where one parish or a diocese could verify someone's sacramental status for prerequisites, Godparent eligibility, valid priestly ordination, and so on. 

I plan to start this project by making a server in Node.js server that has endpoints that then write, say, a baptismal record, to the Ethereum blockchain.

**Comment by natanael127** [comment]
I've just known the project right now. It's a very good idea. Congratulations.

I have had this ideia also and took some notes about the structure of the protocol and also the structure of the chain, well aligned to your comment @droseblockchain !
I thougth about a signed blockchain, like a network of authorized signatures and signers.
To make it easy to happen I thougth to make it with a hardware security module.

I'm thinking about making my own repository in C language (which I know better), but will be a good pleasure to be discussing with you!

Best regards!

**Comment by DevinSRose** [comment]
Natanael, very cool! Hmm, feel free to make a subfolder here with your own project in C language. That would work fine too and we could have it all here. Does that work?

**Comment by natanael127** [comment]
Devin, it would be a pleasure participate with you!
So it would be a folder inside the SacramentalRegistry repository?
Today I'm looking for some util dependencies to be used. I'm also going to make an open document in latex together.
Despite of using C language (more familiar to me), I will be using the JSON format.

**Comment by DevinSRose** [comment]
Yes you can make a sub folder there, that would work just great. I have programmed in C and C++ as well so I will enjoy reading the code in props contributing.

**Comment by natanael127** [comment]
Hey, I've been very busy these last days and I think I won't have time to contribute.
Maybe I'll be in another project of decentralized chat.
I'll be back as soon as possible and will propagate this ideia to my programmer friends.

Best regards,
Thanks for the attention.

**Comment by goznorlives** [comment]
Is this overall effort still active? I just saw https://www.americamagazine.org/politics-society/2019/06/14/can-technology-behind-bitcoin-be-used-build-belief-system and have had similar ideas over the past year or so...

**Comment by DevinSRose** [comment]
goznor, I am not working on it at the moment but anyone can contribute! Are you a developer?

**Comment by goznorlives** [comment]
Yes, I'm a developer. I specialize professionally in Java middleware, but
I'm aware of other related technologies. I have a few hours a day that are
looking for a better home and this sounded worthwhile.

--Fesh
